pkg/kapis/resources/v1alpha2/register.go

```diff
@@ -145,6 +145,9 @@ func AddToContainer(c *restful.Container, k8sClient kubernetes.Interface, factor
 		Param(webservice.QueryParameter("secret", "secret name").
 			Required(false).
 			DataFormat("secret=%s")).
+		Param(webservice.QueryParameter("ignorecert", "whether verify cert if using https repo").
```

Comment:
How about using `insecure` as parameter name which used by [curl](https://curl.se/docs/manpage.html#-k ), or `skipTlsVerify` which used by `kubernetes` 
```
--insecure-skip-tls-verify
--
  | If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
```

Reply:
The rest of the PR is good.

Reply:
fixed